### PR TITLE
1st draft of charter

### PR DESCRIPTION
**Pull request by parispittman**
@cblecker @Phillels @grodrigues3 

top of doc:
This is a WIP. There is jargon in here that will be defined, acronyms drawn out, and links correctly applied after all content suggestions and changes come through.

### CODE REVIEW

**Comment by castrojo** [comment]
> Lazy consensus with a time box of at least 72 hours

Nit: We probably mean 72 working hours to account for weekends and holidays.

**Comment by spiffxp** [comment]
/wip
since this says it's a wip

File: sig-contributor-experience/charter.md [hunk 1]
```diff
@@ -1,58 +1,93 @@
...
+- Strategize, build, and execute on scalable mentoring programs for all contributor levels.
+- Create best practices, trainings, policies, and moderation of Kubernetes communication platforms. This includes user guidelines when the contributor is the user of the platform. Moderation is an always on, never off service that we supply with trusted contributors spanning time zones. This includes immediate action when dealing with code of conduct issues in a public space that are defined below:
+    - All mailing lists/googlegroups where we have ownership, including k-dev@. At least one Owner of the mailing list must be a communications subproject OWNER. [link to mailing list section of moderation guidelines]
+    - Zoom via CNCF funding [link to zoom guidelines]
+    - YouTube via the /kubernetescommunity channel [link]
+    - Slack via CNCF funding [link]
+    - discuss.kubernetes.io via CNCF funding [link]
+- Help onboard new contributors and current into the culture, workflow, and CI of our contributor experience through the constant addition, modification and retirement of contributor documentation and programs tailored to onboarding.
```
Comment (spiffxp) [inline sig-contributor-experience/charter.md]: "... of our contributor experience via the [contributor guide], [contributor summits] and other programs tailored to onboarding

File: sig-contributor-experience/charter.md [hunk 2]
```diff
@@ -1,58 +1,93 @@
...
+- Perform issue triage and maintain the kubernetes/community repository.
```
Comment (spiffxp) [inline sig-contributor-experience/charter.md]: Do mean issue triage across the whole project? Or just the kubernetes/community repo?

eg: "Maintaining the kubernetes/community repository, including issue triage"

File: sig-contributor-experience/charter.md [hunk 3]
```diff
@@ -1,58 +1,93 @@
...
+    - Zoom via CNCF funding [link to zoom guidelines]
+    - YouTube via the /kubernetescommunity channel [link]
+    - Slack via CNCF funding [link]
+    - discuss.kubernetes.io via CNCF funding [link]
+- Help onboard new contributors and current into the culture, workflow, and CI of our contributor experience through the constant addition, modification and retirement of contributor documentation and programs tailored to onboarding.
+- Perform issue triage and maintain the kubernetes/community repository.  
+- Help SIGs with being as transparent and open as possible through creating best practices, guidelines, and general administration of YouTube, Zoom, and our mailing lists where applicable
+- Assist SIGs/WGs with organizational management operations as laid out in the sig governance doc [link]
+- Plan and execute on face to face Contributor Summit(s) and our own SIG Face to Face meetings as well as others (upon request).
+- Own and execute on other events that are targeted to the Kubernetes contributor such as the Steering Committee Election process.
```
Comment (spiffxp) [inline sig-contributor-experience/charter.md]: You could maybe roll a few things up under this bullet, e.g.

"Own and execute events that are targeted to the Kubernetes contributor community, including:
- The weekly Kubernetes Community meeting
- Contributor Summits
- SIG Contributor Experience F2F meetings
- Steering Committee elections (though we do not own policy creation, see 'out of scope' below)
- Retrospective moderation for other SIGs upon request
- other events upon request and consideration"

File: sig-contributor-experience/charter.md [hunk 4]
```diff
@@ -1,58 +1,93 @@
...
+- Distribute contributor related news on various Kubernetes channels and liaison to CNCF for posting blogs and social media on their channels.
+- Work with CNCF to improve DevStats and increase adoption within our community to establish metrics that can be used in process improvements, mapping trends, or charting successes.
```
Comment (spiffxp) [inline sig-contributor-experience/charter.md]: This seems like over-specifying on devstats.  Maybe rephrase in terms of project health? e.g.

"Establish and share metrics that can be used to gauge project health, community health, and general trends, including:
- ongoing work with the CNCF to improve devstats
-

File: sig-contributor-experience/charter.md [hunk 5]
```diff
@@ -1,58 +1,93 @@
...
+
+- Code for the testing and CI infrastructure - that’s SIG Testing
+- k/community ownership of folders for KEPs and Design Proposals. Members are to follow those folders owners files and SIG leadership for the specific issue/PR in question.
+- User community management. We hold office hours because contributors are a large portion of the volunteers that run that program.
+- The contributor experience for repos not included in the [GitHub Management subprojects list](https://git.k8s.io/community/github-management)
+- Steering committee election policy updates and maintenance.
+- We do not create SIGs/WGs but can assist in the various community management needs of their micro communities that would kick off their formation and keep them going.
+- communication platforms that are out of our scope for maintenance and support but we may still have some influence:
+    - r/kubernetes
+    - kubernetesio@ twitter account
```
[the remark this answers is not in the document]
Comment (parispittman) [reply sig-contributor-experience/charter.md]: we don't own it. good catch.

**Comment by nikhita** [comment]
> @spiffxp: failed to re-open PR: state cannot be changed. The pull request cannot be reopened.

Fwiw, it's nice to see that prow explicitly adds a comment for this and does not fail silently.
